core/io/resource_format_binary.cpp

```diff
@@ -2220,7 +2275,18 @@ Error ResourceFormatSaverBinaryInstance::save(const String &p_path, const Ref<Re
 		String res_path = save_order[i]->get_path();
 		res_path = relative_paths ? local_path.path_to_file(res_path) : res_path;
 		save_unicode_string(f, res_path);
-		ResourceUID::ID ruid = ResourceSaver::get_resource_id_for_path(save_order[i]->get_path(), false);
```

Comment:
I don't understand this logic, if this is a subresource it should not be able to be saved standalone.